package.json

```diff
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "101": "^1.2.0",
+    "bluebird": "^3.3.5",
```

Comment:
hmm not sure if there was a reason we did not start by using promises
@bkendall ?